about.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone project module 1</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <body>
+        <section class="header-section">
+            <ul class="top-list-header">
+                <li class="top-list"><img src="img/facebook.png" alt="facebook picture"></li>
+                <li class="top-list"><img src="img/twitter.png" alt="twitter picture"></li>
+                <li class="top-list">English</li>
+                <li class="top-list">My page</li>
+                <li class="top-list">Logout</li>
+            </ul>
+            <header>
+                <div class="main-container">
+                    <div class="header-container">
+                        <nav class="header">
+                            <div class="logo">
+                                <a href="index.html"><img class="my-logo" src="img/logo.png" alt="logo picture"></a>
+                            </div>
+                            <div class="hamburger">
+                                <div class="bar"></div>
+                                <div class="bar"></div>
+                                <div class="bar"></div>
+                            </div>
+                            <ul class="nav-list">
+                                <li class="nav-item about"><a href="index.html">Home</a></li>
+                                <li class="nav-item"><a href="#section1">Program</a></li>
+                                <li class="nav-item"><a href="#section2">Join</a></li>
+                                <li class="nav-item"><a href="#section3">Sponsor</a></li>
+                                <li class="nav-item"><a href="#section4">News</a></li>
+                                <li class="nav-item"><a class="campaign" href="#">Campaign</a></li>
+                            </ul>
+                        </nav>
+                    </div>
+                </div>
+            </header>
+        </section>
+    
+        <main>
+            <section class="title-section">
+                <div class="main-container">
+                    <div class="page-title2">
+                        <article>
+                            <h3 class="greeting">"Hello! Learn about Photography"</h3>
+                            <h1 class="intro">CREATIVE ANNUAL GLOBAL LEARNING SUMMIT</h1>
```

Comment:
**OPTIONAL** - Please note that in order to improve the semantics of your page, I advise that your `hx` tag follow an ascending hierarchy i.e an `h2` tag should only be used after an `h1` tag. I suggest using a `p` tag in place of the `h3` tag. Please fix this 😄